libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -880,6 +880,44 @@ namespace SohImGui {
                     EnhancementSliderInt("King Zora Speed: %dx", "##WEEPSPEED", "gMweepSpeed", 1, 5, "");
                     EnhancementSliderInt("Vine/Ladder Climb speed +%d", "##CLIMBSPEED", "gClimbSpeed", 0, 12, "");
 
+                    const char* autotunic_cvar = "gAutoTunics";
+                    {
+                        int val = CVar_GetS32(autotunic_cvar, 0);
+                        val = MAX(MIN(val, 3), 0);
+                        int fps = val;
```

Comment:
Don't need this fps variable that I assume is a copy-paste leftover.

Reply:
Thanks, fixed and pushed!